tests/datadriven/test_asserts.py

```diff
@@ -0,0 +1,105 @@
+"""
+Tests the datadriven assert methods
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import tests.datadriven as datadriven
+
+
+class TestAsserts(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Ah, I see, Can't we just have default values for the constructor of `DataDrivenTest` rather than unpacking the args tuple?